packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -1,21 +1,27 @@
-import { IDesignSystem } from "../design-system";
+import designSystemDefaults, { IDesignSystem } from "../design-system";
 import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
 import { applyLocalizedProperty, toPx } from "@microsoft/fast-jss-utilities";
 import { typeRamp } from "../utilities/typography";
 import { IToggleClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { isEmpty } from "lodash-es";
 import * as Chroma from "chroma-js";
 import { Direction } from "@microsoft/fast-application-utilities";
 
 function setFloatAndClear(clearFloat: boolean = true): ICSSRules<IDesignSystem> {
     return {
-        float: (config: IDesignSystem): string => config.direction === Direction.ltr ? "left" : "right",
-        clear: clearFloat ? ((config: IDesignSystem): string => config.direction === Direction.ltr ? "left" : "right") : null
+        float: (config: IDesignSystem): string => {
+            return (isEmpty(config) || config.direction === Direction.ltr) ? "left" : "right";
+        },
+        clear: (config: IDesignSystem): string => {
+            return clearFloat ? (isEmpty(config) || config.direction === Direction.ltr ? "left" : "right") : null;
+        }
     };
 }
 
-/* tslint:disable:max-line-length */
+/* tslint:disable-next-line */
 const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IToggleClassNameContract, IDesignSystem> => {
-/* tslint:enable:max-line-length */
+    config = isEmpty(config) ? designSystemDefaults : config;

```

Comment:
Again, I don't think we should be re-assigning a passed value. We should assign this as safely as possible so using a new variable is fine.

Reply:
Let's land the convo above prior to any follow-up on this